docs/DeveloperGuide.md

```diff
@@ -118,15 +120,16 @@ For each of the five components:
 * Its API is defined in an `interface` with the same name as the Component.
 * Its functionality is exposed using a concrete `{Component Name} Manager` class which implements the corresponding API
 `interface` mentioned in the previous point.
-    - For example, the `Logic` component defines its API in the `Logic.java` interface and exposes its functionality
+    - For example, the `Logic` component (see the class diagram below) defines its API in the `Logic.java` interface and exposes its functionality
     using the `LogicManager.java` class which implements the `Logic` interface.
 
 ![Class Diagram of the Logic Component](images/LogicClassDiagram.png)
```

Comment:
I think for the class diagram we should include state, and also for delete 1 need to mention whether we are talking about delete 1 under context or expenditure or budget!